content/docs/state-and-lifecycle.md

```diff
@@ -68,33 +69,33 @@ ReactDOM.render(
 );
 ```
 
-To implement this, we need to add "state" to the `Clock` component.
+इसे लागू करने के लिए, हमें `Clock` कौम्पोनॅन्ट में "state" को `Clock` में डालना होगा।
 
-State is similar to props, but it is private and fully controlled by the component.
+State "props" के समान है, लेकिन यह निजी और कौम्पोनॅन्ट द्वारा पूरी तरह से नियंत्रित है।
 
-We [mentioned before](/docs/components-and-props.html#functional-and-class-components) that components defined as classes have some additional features. Local state is exactly that: a feature available only to classes.
+हमने [पहले उल्लेख किया है](/docs/components-and-props.html#functional-and-class-components) कि क्लासेज के रूप में परिभाषित कौम्पोनॅन्टों में कुछ अतिरिक्त विशेषताएं हैं। लोकल state ठीक यही है: एक सुविधा जो केवल क्लासेज के लिए उपलब्ध है।
```

Comment:
कौम्पोनॅन्टों -> components

Reply:
लोकल state ठीक यही है:  incorrect translation

Reply:
I meant components in hindi here, not in english

Reply:
> लोकल state ठीक यही है: incorrect translation

Already corrected